docs/explanations/using_pro_offline.rst

```diff
@@ -0,0 +1,68 @@
+Using Ubuntu Pro in an offline environment
+******************************************
+
+Airgapping (or air gapping) is a security measure that involves isolating a
+machine or network from external/unsecured networks to protect sensitive data.
+Airgapped systems usually operate “internetless”, or with limited network
+connectivity. 
+
+Ubuntu Pro can still be used in an offline environment, via an airgapped
+contract server and local mirrors. This allows Ubuntu Pro tokens to be
+distributed across the isolated network, and provides the security support and
+other benefits of Pro even without internet access.
+
+How do I sync Pro packages in an airgapped system?
+==================================================
+
+This is an advanced use-case, and our `Customer Support teams`_ can help you
+implement this in the best way for your specific environment. Note that this
+requires a paid subscription to Ubuntu Pro.
+
+What tools do I need?
+=====================
+
+Updating Ubuntu systems in an offline environment requires mirroring Ubuntu
+repositories. Since Ubuntu packages can be consumed via deb packages and/or
+snaps, the tool needed to mirror Ubuntu repositories depends on where you
+source your packages from.
+
+Mirror deb repositories with Landscape
+--------------------------------------
+
+Landscape is a management tool that can host a mirror of Ubuntu deb
+repositories. 
+
+In future release cycles, Landscape will also include features specific to
+airgapped use-cases. You can keep up to date with these developments in the
+`Landscape Beta Discourse`_.
+
+Mirror snaps with Snap store proxy
+----------------------------------
+
+The Snap store proxy is the equivalent tool for hosting the snap packages that
+your organisation is consuming.
+
+`Snap-store-proxy`_ is a snap that provides an edge proxy to the Snap store, so
+that after your device has been registered with the proxy, all communications
+with the Snap store will be sent via the proxy.
+ 
+Security patching
+=================
+
+Livepatch is the tool that provides patching for critical and high security
+vulnerabilities in the Ubuntu kernel. It can also be used in an airgapped
+setup using Livepatch on-prem, which our `Customer Support teams`_ can help you
+to set up.
```

Comment:
I'm not sure if this part is true - I think livepatch-on-prem requires a network connection to the Canonical livepatch server. We should double check with livepatch team before promising this.

Reply:
I said airgapped when I think I should have said "firewalled", which is the language they use in the livepatch on-prem docs.